book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person)
+    Rental.new(person, self)
+  end
```

Comment:
- [ ] @laiifuu great work implementing the changes requested but as illustrated in the previous reviewer's example your code is missing the date, sorry for the error  @Munsa1 made when writing the review,
![image](https://user-images.githubusercontent.com/61408860/208977786-058b3633-75ac-45fb-acd2-159c9c9664a2.png)

Reply:
Hi @danmainah, the reason I omitted the date parameter is that in Rental's initialize method, the @date gets assigned a new date instead of one passed as a parameter. should I change the initialize method as well?

Reply:
@laiifuu no you don't need to change the initialize method since we are using the Rental initialized date.